### PR TITLE
📖 Adds kustomize substitutions for vSphere

### PR DESCRIPTION
**Pull request by srm09**
**What this PR does / why we need it**:
Adds kustomize substitutions for tilt development with vSphere.

**Which issue(s) this PR fixes**:
Fixes n/a

### CODE REVIEW

**Comment by srm09** [comment]
@ykakarap thanks for the catch, wonder why the markdown linter did not catch this.

**Review by killianmuldoon — APPROVED** [review]
/lgtm

File: docs/book/src/developer/tilt.md [hunk 1]
```diff
@@ -169,6 +169,15 @@ kustomize_substitutions:
   GCP_B64ENCODED_CREDENTIALS: "your credentials here"
 ```
 
+{{#/tab }}
+{{#tab vSphere}}
+
+```yaml
+kustomize_substitutions:
+  "VSPHERE_USERNAME": "administrator@vsphere.local",
+  "VSPHERE_PASSWORD": "Admin123",
```
Comment (sbueringer) [inline docs/book/src/developer/tilt.md:178]:
```suggestion
  VSPHERE_USERNAME: "administrator@vsphere.local"
  VSPHERE_PASSWORD: "Admin123"
```
Up to you if you want to drop the quotes around the keys, but the commas have to go :)

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Pull request merged**